examples/dump_replay.lua

```diff
@@ -0,0 +1,100 @@
+require 'torch'
```

Comment:
Actually, we should maybe include a replay in the repo 😮 This way your `th dump_replay.lua -t $VM_IP` can be complemented with a specific replay setup for the sake of the example.